doc/how_to_guides/isaac_panda/launch/python.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright 2022 PickNik Inc.
+# All rights reserved.
+#
+# Unauthorized copying of this code base via any medium is strictly prohibited.
+# Proprietary and confidential.
+
+# Script to call the isaac launch system (see python.sh in the ISAAC_SCRIPT_DIR)
+
+ISAAC_SCRIPT_DIR="$HOME/.local/share/ov/pkg/isaac_sim-2022.2.0"
```

Comment:
Maybe add a note saying this is where you change things if you have a different version / Isaac Sim is installed elsewhere.

Reply:
I thought about that but don't want this tutorial to have to support a bunch of different configurations. As is it's pretty easy for the user to switch the path and test it out. If you guys think this is worth it I can add the comment.